dotnet-core-music-linux/azuredeploy.json

```diff
@@ -0,0 +1,435 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmName": {
```

Comment:
yes - the scenario you want to demonstrate doesn't really benefit from a user defined resource name...  one less question to ask, one less thing to fail...